src/ensembl/src/header/launchbar/Launchbar.scss

```diff
@@ -82,6 +82,9 @@
     fill: $white;
     background-color: $grey;
   }
+  button {
+    cursor: default;
+  }
```

Comment:
What do we need this for?

Reply:
Thats because we have pointer for disabled launchbar buttons.